dialog/src/main/java/roboy/dialog/states/devoStates/QuestionRoboyQAState.java

```diff
@@ -65,8 +62,7 @@
     private final static String TRANSITION_FINISHED_ANSWERING = "finishedQuestionAnswering";
     private final static String TRANSITION_LOOP_TO_NEW_PERSON = "loopToNewPerson";
     private final static String TRANSITION_LOOP_TO_KNOWN_PERSON = "loopToKnownPerson";
-    private final static String TRANSITION_SWITCH_TO_GAMING = "switchToGaming";
-    private final static int MAX_NUM_OF_QUESTIONS = 5;
+    private final static int MAX_NUM_OF_QUESTIONS = 6;
```

Comment:
why is `TRANSITION_SWITCH_TO_GAMING` now removed?

Reply:
Copy paste from Devel, took https://github.com/Roboy/roboy_dialog/blob/devel/dialog/src/main/java/roboy/dialog/states/devoStates/QuestionRoboyQAState.java because I know that works.

I can revert this and only copy paste the bare minimum to get the APIs to work

Reply:
From what I can tell, we roboy.dialog.states.ordinaryStates.QuestionAnsweringState and used that as a basis to create the current Devo class. The Devo personality was written to be a "demo", that would answer 6 questions before doing the Rick and Morty reference at the end. APIs would be demoed in these 6 questions.

Since a game would "eat up" one of the questions, these were disabled. 


Question Answering State (for Ordinary Personality) was not touched afaik.